app/views/RoomMembersView/index.tsx

```diff
@@ -385,9 +385,9 @@ const RoomMembersView = (): React.ReactElement => {
 			<FlatList
 				data={filteredMembers || state.members}
 				renderItem={({ item }) => (
-					<View style={{ backgroundColor: colors.surfaceRoom }}>
+					<View style={{ backgroundColor: colors.surfaceRoom, marginBottom: 0.5 }}>
```

Comment:
Can you do it on a separate PR?
This is a known issue on our app and we haven't been able to fix it.
https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/5879

Can you give it a try?
Thanks!

Reply:
Sure 😊

Reply:
Sure 😊